src/components/ResultsView.tsx

```diff
@@ -286,11 +314,20 @@ const ResultsView: ResultsViewComponent = ({
       );
   };
   const debounceRef = useRef(0);
+  const [showInterface, setShowInterface] = useState(
+    layout.interface !== "hide"
+  );
+  const [showMenuIcons, setShowMenuIcons] = useState(
+    layout.interface !== "hide"
```

Comment:
both showMenIcons and setInterface are being source from the same persistent state, which either means it should be stored separately or we have redundant local state

Reply:
Sorry, I don't understand.

Are you suggesting we should store `layout.interface` in a `useState` hook as well??

Both `showMenuIcons` and `showInterface` use the `layout.interface` for their initial state, but their states change depending on different events.  (Eg, `showMenuIcons` updates on hover)

Reply:
In which case, it seems like `showMenuIcons` is actually not dependent on the value of `layout.interface`, since we check them both independently on `L378`, so it should probably just be initialized as `useState(false)`.

This is non-blocking feedback though, will continue to merge